app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>ConferenceApp</title>
+    <title><%= page_title(yield(:page_title)) %></title>
```

Comment:
Calling `content_for` for page_title in individual page configures the page title like https://github.com/kaigionrails/conference-app/pull/256/files#diff-d355e12f3e1517389297b283028f8ef3dde4e9d51127d3fbb8f57924b8770f29R1